gearcatalog/frontend/src/actions/actions.types.ts

```diff
@@ -1,2 +1,21 @@
+import { Gear } from '../components/gear/types';
+
+type GearNoId = {
```

Comment:
What is the use case for this type?

Reply:
Adding new gear.

Reply:
Could you document its use in the code? As it is it seems a bit odd.

Reply:
I need a gear with no id when adding new gear to the database. I went ahead and moved it to the gear types file and added a comment for it explaining why it's there.